Makefile

```diff
@@ -468,12 +482,22 @@ release: ## Re-generate generated code and run tests
 	$(MAKE) --no-print-directory test
 
 build: ## Build all packages and all tests (ensures everything compiles)
-	$Q echo 'Building all packages...'
+	$Q echo 'Building all packages and submodules...'
 	$Q go build ./...
+	$Q cd common/archiver/gcloud; go build ./...
+	$Q cd cmd/server; go build ./...
```

Comment:
if/when we get a ton of these, this is pretty easy to template.  not worth it for 3 though IMO, make is pretty obtuse